examples/common/python/shared_kv/remote_lmdb/lmdb_test_app.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2019 Intel Corporation
```

Comment:
Please rename file as test_lmdb_app.py. Naming convention followed for test files is test_xxx

Reply:
Done.